### PR TITLE
GameDB: Add vuClampMode fix to EA Burnout titles.

### LINKED ISSUE #2047: Burnout 3 - Garage Textures Flickering
***PCSX2 version:***
1.5.0-2156

***PCSX2 options:***
All defaults.

***Plugins used:***
GSDX OpenGL Software (AVX2)

All default options otherwise.

***Description of the issue:***
If you look at the vents and pipes, you'll see they flicker a white/grey colour.

***How to reproduce the issue:***
1. Start the game, save profile etc.
2. Go to time attack, and select a course/track
3. You'll be in the garage, look at the vents and pipes near the top of the screen.

***Last known version to work:***
Issue has always been present.

***PC specifications:***
CPU: i5-6400
GPU: Intel HD 530 Graphics (I've gotten this same issue using a 7970.)
RAM: 8GB
OS: Windows 10 x64

PCSX2:
gsdx_20170814154423

Console (Xbox, the garage is the same on both platforms):
image4

GS Dump: [b3.zip](https://github.com/PCSX2/pcsx2/files/1237635/b3.zip)

### PR DESCRIPTION
**Pull request by UlsterRose**
This fixes bugs with lighting in various places across the three EA Burnout games. Fixes #2047.

### CODE REVIEW

File: bin/GameIndex.dbf [hunk 1]
```diff
@@ -38751,6 +38766,7 @@ Serial = SLUS-21050
 Name   = Burnout 3 - Takedown
 Region = NTSC-U
 Compat = 5
+vuClampMode = 0 //Fixes buggy lighting on certain objects
```
[the remark this answers is not in the document]
Comment (UlsterRose) [reply bin/GameIndex.dbf]: What are you referring to?

[the remark this answers is not in the document]
Comment (tony971) [reply bin/GameIndex.dbf]: I see that this serial is identified as both "Medal of Honor - Frontline" and "Burnout 3 - Takedown." Would there be unintended consequences of applying this to both?

[the remark this answers is not in the document]
Comment (UlsterRose) [reply bin/GameIndex.dbf]: That's just the way GitHub shows the changes. If you look at the raw file, they are separate as they should be.

**Comment by ssakash** [comment]
@PCSX2/bug-squad - Anyone tested this?

**Comment by seapancake** [comment]
Can confirm the pipes render correctly now, not able to attach screenshot for some reason.

**Comment by ssakash** [comment]
> Can confirm the pipes render correctly now, not able to attach screenshot for some reason.

Thanks for testing, I looked at the ones you provided at Discord.

**Pull request merged**